### PR TITLE
OTG Test gNMI-1.10 update

### PR DESCRIPTION
**Pull request by octpetre**
Updating with changes to the ATE test in https://github.com/openconfig/featureprofiles/pull/395

### CODE REVIEW

**Comment by octpetre** [comment]
@liulk could you please assign? Thanks

**Review by sezhang2 — APPROVED** [review]

File: feature/gnmi/otg_tests/telemetry_basic_check_test/telemetry_basic_check_test.go [hunk 1]
```diff
@@ -39,6 +40,7 @@ const (
 	adminStatusDown = telemetry.Interface_AdminStatus_DOWN
 	operStatusUp    = telemetry.Interface_OperStatus_UP
 	operStatusDown  = telemetry.Interface_OperStatus_DOWN
+	maxPortVal      = "FFFFFEFF" // Maximum Port Value : https://github.com/openconfig/public/blob/2049164a8bca4cc9f11ffb313ef25c0e87303a24/release/models/p4rt/openconfig-p4rt.yang#L63-L81
```
Comment (liulk) [inline feature/gnmi/otg_tests/telemetry_basic_check_test/telemetry_basic_check_test.go:43]:
```suggestion
	// Maximum Port Value: https://github.com/openconfig/public/blob/2049164a8bca4cc9f11ffb313ef25c0e87303a24/release/models/p4rt/openconfig-p4rt.yang#L81
	maxPortVal      = 0xFFFFFEFF
```

File: feature/gnmi/otg_tests/telemetry_basic_check_test/telemetry_basic_check_test.go [hunk 2]
```diff
@@ -593,6 +595,11 @@ func TestP4rtInterfaceID(t *testing.T) {
 	d := &telemetry.Device{}
 	i := d.GetOrCreateInterface(dp.Name())
 
+	maxPortValDec, err := strconv.ParseUint(maxPortVal, 16, 32)
+	if err != nil {
+		t.Fatalf("Error while converting portID value: %v", err)
+	}
+
```
Comment (liulk) [inline feature/gnmi/otg_tests/telemetry_basic_check_test/telemetry_basic_check_test.go:602]:
```suggestion
```

File: feature/gnmi/otg_tests/telemetry_basic_check_test/telemetry_basic_check_test.go [hunk 3]
```diff
@@ -601,10 +608,10 @@ func TestP4rtInterfaceID(t *testing.T) {
 		portID: 1,
 	}, {
 		desc:   "AvePortID",
-		portID: math.MaxUint32 / 2,
+		portID: uint32(maxPortValDec) / 2,
```
Comment (liulk) [inline feature/gnmi/otg_tests/telemetry_basic_check_test/telemetry_basic_check_test.go:611]:
```suggestion
		portID: uint32(maxPortVal) / 2,
```

File: feature/gnmi/otg_tests/telemetry_basic_check_test/telemetry_basic_check_test.go [hunk 4]
```diff
@@ -601,10 +608,10 @@ func TestP4rtInterfaceID(t *testing.T) {
...
 	}, {
 		desc:   "MaxPortID",
-		portID: math.MaxUint32,
+		portID: uint32(maxPortValDec),
```
Comment (liulk) [inline feature/gnmi/otg_tests/telemetry_basic_check_test/telemetry_basic_check_test.go:614]:
```suggestion
		portID: uint32(maxPortVal),
```

**Review by liulk — CHANGES_REQUESTED** [review]

File: feature/gnmi/otg_tests/telemetry_basic_check_test/telemetry_basic_check_test.go (hunk 1 above)
Comment (octpetre) [reply feature/gnmi/otg_tests/telemetry_basic_check_test/telemetry_basic_check_test.go:43]: Done

File: feature/gnmi/otg_tests/telemetry_basic_check_test/telemetry_basic_check_test.go (hunk 3 above)
Comment (octpetre) [reply feature/gnmi/otg_tests/telemetry_basic_check_test/telemetry_basic_check_test.go:611]: Done

File: feature/gnmi/otg_tests/telemetry_basic_check_test/telemetry_basic_check_test.go (hunk 2 above)
Comment (octpetre) [reply feature/gnmi/otg_tests/telemetry_basic_check_test/telemetry_basic_check_test.go:602]: Done

**Review by liulk — APPROVED** [review]

**Pull request merged**